packages/opds-web-client/src/hooks/useDownloadButton.ts

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
```

Comment:
There should be tests for this hook.

Reply:
I know these are indirectly tested in components, but explicit tests would be nice.